code/__HELPERS/time.dm

```diff
@@ -7,6 +7,11 @@
 
 #define TICKS *world.tick_lag
 
+#define SECONDS_TO_LIFE_CYCLES /2
```

Comment:
Not sure how I feel about making the Life controller use an explicit wait time. I think it's better to have the subsystem use whatever inherited value (which is 20 deciseconds) so that changes made to that apply to all inheriting subsystems.

Reply:
For reference, on /tg/ the Life controller has explicit 20 deciseconds wait: https://github.com/tgstation/tgstation/blob/2c842bc9073824fe464ee8c4470276a43a7ff21d/code/controllers/subsystem/mobs.dm#L6
I'm not sure we need to couple Life to the default subsystem wait times, since Life is such a prominent subsystem already. Everyone knows Life tick is 2 seconds, but very few realize that's also default for other controllers.

If you want to keep Life and default decoupled then what about:
```dm
#define DEFAULT_CONTROLLER_WAIT_DURATION 2 SECONDS
#define LIFE_CYCLE_WAIT_DURATION DEFAULT_CONTROLLER_WAIT_DURATION 
```
and set the default controller wait to `DEFAULT_CONTROLLER_WAIT_DURATION` (with or without also setting it on Life)?

Reply:
I deem this fine. If you really want to, you *may* be able to make this
```dm
#define SECONDS_TO_LIFE_CYCLES SSmobs.wait / 10
```
But rules of constant expressions may forbid this